src/azure-cli/azure/cli/command_modules/network/custom.py

```diff
@@ -5043,6 +5043,10 @@ def update_nw_flow_log(cmd,
         if not workspace:
             raise CLIError('Name or ID of workspace is invalid')
 
+        if instance.flow_analytics_configuration.network_watcher_flow_analytics_configuration is None:
```

Comment:
Does flow_analytics_configuration always exist? If not, instance.a.b would raise exception.

Reply:
No. I tested